packages/config/src/proxy.js

```diff
@@ -54,6 +73,18 @@ function createInsightsProxy({ betaEnv, rootFolder, localChrome, customProxy = [
                     }
                 }
             }] : []),
+            ...proxyRoutes ? Object.entries(proxyRoutes).map(([ route, redirect ]) => {
+                const currTarget = redirect.host || redirect;
+                return {
+                    context: (path) => path.includes(route),
+                    target: currTarget === 'PORTAL_BACKEND_MARKER' ? target : currTarget,
+                    secure: false,
+                    changeOrigin: true,
+                    autoRewrite: true,
+                    ws: true,
+                    ...redirect
```

Comment:
add a check if it's a object (otherwise the string will be spread) and we should also omit the host property from it.